src/Libraries/RevitNodes/Elements/ElevationMarker.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using Autodesk.Revit.DB;
+using Revit.Elements.Views;
+using Revit.GeometryConversion;
+using RevitServices.Transactions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Revit.Elements
+{
+    public class ElevationMarker : Element
+    {
+        #region Internal properties
+
+        /// <summary>
+        /// An internal handle on the ElevationMarker
+        /// </summary>
+        internal Autodesk.Revit.DB.ElevationMarker InternalMarker
+        {
+            get; private set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        [SupressImportIntoVM]
+        public override Autodesk.Revit.DB.Element InternalElement
+        {
+            get { return InternalMarker; }
+        }
+
+        #endregion
+
+        #region Private constructors
+
+        /// <summary>
+        /// Private constructor
+        /// </summary>
+        private ElevationMarker(Autodesk.Revit.DB.ElevationMarker marker)
+        {
+            SafeInit(() => InitElevationMarker(marker));
+        }
+
+        #endregion
+
+        #region Helpers for private constructors
+
+        /// <summary>
+        /// Initialize a ElevationMarker element
+        /// </summary>
+        private void InitElevationMarker(Autodesk.Revit.DB.ElevationMarker marker)
+        {
+            InternalSetElevationMarker(marker);
+        }
+
+        #endregion
+
+        #region Private mutators
+
+        /// <summary>
+        /// Set the InternalMarker property and the associated element id and unique id
+        /// </summary>
+        /// <param name="marker"></param>
+        private void InternalSetElevationMarker(Autodesk.Revit.DB.ElevationMarker marker)
+        {
+            InternalMarker = marker;
+            InternalElementId = marker.Id;
+            InternalUniqueId = marker.UniqueId;
+        }
+
+        #endregion
+
+        #region Public static constructors
+
+        /// <summary>
+        /// Creates a new ElevationMarker.
+        /// </summary>
+        /// <param name="viewFamilyType">The ViewFamilyType that will be used by all elevations hosted on the new ElevationMarker.</param>
+        /// <param name="location">The desired origin for the ElevationMarker.</param>
+        /// <param name="initialViewScale">The view scale will be automatically applied to new elevations created on the ElevationMarker. The scale is the ratio of true model size to paper size.</param>
```

Comment:
provide an example input value (& what it means) at the end. like `100` input for `1:100` scale.